cpp/src/rolling/jit/code/kernel.cpp

```diff
@@ -26,33 +26,29 @@ const char* kernel_headers =
 R"***(
 #include <cudf/types.hpp>
 #include <cudf/utilities/bit.hpp>
+#include <../src/rolling/rolling_jit_detail.hpp>
```

Comment:
What about adding this to the JIT'd `operation.h` string in `src/rolling/operation.cpp`?

Reply:
@trxcllnt I am not sure if I understand. The `operation_h` string in `cpp/src/rolling/jit/code/operation.cpp` is a hard coded code string which is not stringified at compile time. The code in `cpp/src/rolling/rolling_jit_detail.hpp` is to be stringified at compile time.

Reply:
I can certainly also hard code the `___src_rolling_rolling_jit_detail_hpp` string but it has the drawback @devavret mentioned above.

Reply:
Oh my mistake, I forgot we changed how JIT headers are included.